homeassistant/components/rainbird/sensor.py

```diff
@@ -61,7 +61,10 @@ def name(self):
     @property
     def unit_of_measurement(self):
         """Return the units of measurement."""
-        return self._unit_of_measurement
+        if isinstance(self._unit_of_measurement, list):
+            return self._unit_of_measurement[int(self._state == 1)]
+        else:
+            return self._unit_of_measurement
```

Comment:
A unit of measurement cannot be dynamic.
Please revert this change.

Reply:
Would a simple **d** as unit be okay?

Reply:
We generally use "days" as the unit in this case, so the current was correct.
Changing units is a breaking change as well, so unless there is a good reason to break it, I suggest we don't.

Reply:
Why did you still use `d`, while I just said, we use "days"?

Reply:
😭 Sorry did the change before i saw your response. Reverted it.